packages/trader/src/Stores/client-store.js

```diff
@@ -975,6 +980,29 @@ export default class ClientStore extends BaseStore {
         this.device_data = device_data;
     }
 
+    @action.bound
+    onSetResidence({ residence }, cb) {
```

Comment:
why use a callback instead of a promise?

Reply:
No particular reason, wanted to be consistent with `onAccountSignUp`.